README.md

```diff
@@ -8,283 +8,46 @@ Yocto BSP layer for the Raspberry Pi boards - http://www.raspberrypi.org/ .
-|                | 256MB RP. Overrides gpu_mem. Max 448. Default not set.              |
-| `GPU_MEM_1024` | GPU memory in megabyte for the 1024MB Raspberry Pi. Ignored by the  |
-|                | 256MB/512MB RP. Overrides gpu_mem. Max 944. Default not set.        |
-
-#### 3.C. Add purchased license codecs
-To add you own licenses use variables `KEY_DECODE_MPG2` and `KEY_DECODE_WVC1` in local.conf. Example:
-```
-KEY_DECODE_MPG2 = "12345678"
-KEY_DECODE_WVC1 = "12345678"
-```
-You can supply more licenses separated by comma. Example:
-```
-KEY_DECODE_WVC1 = "0x12345678,0xabcdabcd,0x87654321"
-```
-
-#### 3.D. Disable overscan
-By default the GPU adds a black border around the video output to compensate for TVs which cut off part of the image. To disable this set this variable in local.conf:
-`DISABLE_OVERSCAN = "1"`
-
-#### 3.E. Set overclocking options
-The Raspberry PI can be overclocked. As of now overclocking up to the "Turbo Mode" is officially supported by the raspbery and does not void warranty. Check the config.txt for a detailed description of options and modes. Example turbo mode:
-```
-ARM_FREQ = "1000"
-CORE_FREQ = "500"
-SDRAM_FREQ = "500"
-OVER_VOLTAGE = "6"
-```
-
-#### 3.F. Video camera support with V4L2 drivers
-Set this variable to enable support for the video camera (Linux 3.12.4+ required)
-`VIDEO_CAMERA = "1"`
-
-#### 3.G. Enable offline compositing support
-Set this variable to enable support for dispmanx offline compositing:
-`DISPMANX_OFFLINE = "1"`
-
-This will enable the firmware to fall back to off-line compositing of Dispmanx elements. Normally the compositing is done on-line, during scanout, but cannot handle too many elements. With off-line enabled, an off-screen buffer is allocated for compositing. When scene complexity (number and sizes
-of elements) is high, compositing will happen off-line into the buffer.
-
-Heavily recommended for Wayland/Weston.
-
-See: http://wayland.freedesktop.org/raspberrypi.html
-
-#### 3.H. Enable kgdb over console support
-To add the kdbg over console (kgdboc) parameter to the kernel command line, set this variable in local.conf:
-`ENABLE_KGDB = "1"`
-
-#### 3.I. Boot to U-Boot
-To have u-boot load kernel image, set in your local.conf:
-`KERNEL_IMAGETYPE = "uImage"`
-
-This will make kernel.img be u-boot image which will load uImage. By default, kernel.img is the actual kernel image (ex. Image).
-
-#### 3.J. Image with Initramfs
-To build an initramfs image :
-* Set this 3 kernel variables (in linux-raspberrypi.inc for example)
-  - kernel_configure_variable BLK_DEV_INITRD y
-  - kernel_configure_variable INITRAMFS_SOURCE ""
-  - kernel_configure_variable RD_GZIP y
-* Set the yocto variables (in linux-raspberrypi.inc for example)
-  - `INITRAMFS_IMAGE = "<a name for your initramfs image>"`
-  - `INITRAMFS_IMAGE_BUNDLE = "1"`
-* Set the meta-rasberrypi variable (in raspberrypi.conf for example)
-  - `KERNEL_INITRAMFS = "-initramfs"`
-
-#### 3.K. Device tree support
```

Comment:
I'd agree, all non-device tree kernel versions are now gone on the master branch and pyro at least. There may be some code we can remove from the kernel recipes/classes as well, I'd need to check.